apache/incubator-zeppelin
# fix bootstrap dialog default button

### What is this PR for?
Fix default focus issue on bootstrap alert/confirm box

### What type of PR is it?
Improvement

### Is there a relevant Jira issue?
no

### How should this be tested?
Click on any action which prompts the user with a confirm box. Press enter/return for OK. Esc or click outside for cancel
